src/components/common/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -131,7 +131,7 @@
           <v-btn large color="primary"
             id="name-translation-done"
             :disabled="isAddingNameTranslation || !hasPendingChange"
-            @click="setNameTranslations()"
+            @click="saveNameTranslations()"
```

Comment:
`saveNameTranslations()` is a local method.

`setNameTranslations()` is a store method.